docs-src/content/_index.md

```diff
@@ -4,7 +4,7 @@ title: "Cubic SDK Documentation"
 
 # Cubic API Overview
 
-Cubic is Cobalt’s automatic speech recognition (ASR) engine. It can be deployed on-prem and accessed over the network or on your local machine via an API. We currently support C++, C#, Go, Java and Python, and can support for more languages as required.
+Cubic is Cobalt’s automatic speech recognition (ASR) engine. It can be deployed on-prem and accessed over the network or on your local machine via an API. We currently support C++, C#, Go, Java and Python, and can add support for more languages as required.
```

Comment:
```suggestion
Cubic is Cobalt’s automatic speech recognition (ASR) engine. It can be deployed on-prem and accessed over the network or on your local machine via an API. We currently support C++, C#, Go, Java, Python and Swift, and can add support for more languages as required.
```